.github/workflows/mobile_job.yml

```
@@ -161,7 +161,7 @@ jobs:
           if [[ "${IPA_ARCHIVE}" == http* ]]; then
             IPA_ARCHIVE_OUTPUT="ci.ipa"
 
-            curl -s "${IPA_ARCHIVE}" -o "${IPA_ARCHIVE_OUTPUT}"
+            curl -s "${IPA_ARCHIVE}" -o "${IPA_ARCHIVE_OUTPUT}" --fail
```

Since we seem to use curl a lot. Maybe we should create an environment variable and simplify the command here so that we don't forget to copy paste the same parameters to all the instances.

At the top of this file:

```
env:
  CURL_PARMS: '-s --fail'
```

Then we can change lines like this to:

```suggestion
curl ${CURL_PARAMS} "${IPA_ARCHIVE}" -o "${IPA_ARCHIVE_OUTPUT}"
```